lima.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: lima
+  version: 0.14.2
+  epoch: 0
+  description: Linux virtual machines, typically on macOS, for running containerd
```

Comment:
Is lima usable/useful from inside a container on (presumably) Linux? I think you know more about lima than I do, are you planning something by supporting this in Wolfi?

Reply:
I'm not sure about that, good question 🤔 @akihirosuda can you please help me ?